canvas_modules/common-canvas/package.json

```diff
@@ -55,7 +63,7 @@
     "@rollup/plugin-babel": "5.3.0",
     "@rollup/plugin-commonjs": "21.0.1",
     "@rollup/plugin-json": "4.1.0",
-    "@rollup/plugin-node-resolve": "13.0.6",
+    "@rollup/plugin-node-resolve": "^13.0.6",
```

Comment:
Any reason you changed the devDependencies?  These should stay static since consuming applications wouldn't install them.